xds/pkg/cluster/cluster.go

```diff
@@ -18,6 +18,20 @@ package cluster
 
 import (
 	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"quilkin.dev/xds-management-server/pkg/metrics"
+
```

Comment:
Super tiny nit -- I'd _love_ to get rid of spaces between the non-go native imports.

Non blocking though, it's my personal irk. So if I get really annoyed I'll go in at a later date and do it myself 😄